requirements-dev

```diff
@@ -1,2 +1,3 @@
 black==22.3.0
```

Comment:
I kept black in case we still use it somewhere. Maybe it can be removed, not sure.

Reply:
I don't see the point. With this change, black is not used anymore, or not? So then it should be removed.

Reply:
No, it should not be used elsewhere. So remove it.